hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hss/DispatchForResponseCodeHssCall.java

```diff
@@ -129,8 +129,10 @@ public DispatchForResponseCodeHssCall(
         var status = recordBuilder.status();
         if (status != SUCCESS) {
             recordBuilder.status(status);
+            return completionWith(gasRequirement, recordBuilder, encodedRc(status));
```

Comment:
I'm not sure if I understand this change.  Does not `resultEncoder.apply` effectively do the same thing as `encodedRC`?